packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -467,6 +516,13 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin[] {
 					})
 				);
 			},
+			async closeBundle() {
```

Comment:
I would really appreciate some sanity check here. According to [vite docs](https://vite.dev/guide/api-plugin.html#universal-hooks) this is the right hook where we'd want to cleanup process resources, 

<img width="542" alt="Screenshot 2025-07-08 at 18 25 45" src="https://github.com/user-attachments/assets/3c39cea3-cbda-49a9-9076-f3a1d0060e90" />

but I'm new to vite so I might be overlooking smth